styles.css

```diff
@@ -166,6 +166,9 @@
 .qtype_crossword-grid-wrapper .wrap-clue .clue-content {
     word-break: break-word;
 }
+.qtype_crossword-grid-wrapper .wrap-clue .clue-count {
+    margin-right: 3px;
+}
```

Comment:
Moodle is trying to move away from custom CSS, and instead use Boostrap classes.

However, this issue might not be a good time to try to change this in qtype_crossword. Perhaps we should make a separate issue to fix that?